Signal/src/ViewControllers/PhotoLibrary/ImagePickerController.swift

```diff
@@ -53,6 +55,23 @@ class ImagePickerGridController: UICollectionViewController, PhotoLibraryDelegat
         navigationItem.leftBarButtonItem = UIBarButtonItem(barButtonSystemItem: .cancel,
                                                            target: self,
                                                            action: #selector(didPressCancel))
+
+        titleLabel.text = photoCollection.localizedTitle()
+        titleLabel.textColor = Theme.primaryColor
+        titleLabel.font = UIFont.ows_dynamicTypeBody.ows_mediumWeight()
+
+        let titleIconView = UIImageView()
+        titleIconView.tintColor = Theme.primaryColor
+        titleIconView.image = UIImage(named: "navbar_disclosure_down")?.withRenderingMode(.alwaysTemplate)
+
+        let titleView = UIStackView(arrangedSubviews: [titleLabel, titleIconView])
+        titleView.axis = .horizontal
+        titleView.alignment = .center
+        titleView.spacing = 5
+        titleView.isUserInteractionEnabled = true
+        titleView.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(titleTapped)))
+        navigationItem.titleView = titleView
```

Comment:
While the details of the design are pending, I've tried to follow our reference.

Reply:
IIRC - You might have some issues with a titleView like this on iOS9/10. Please verify. You can see the MediaPageView titleView as an example if it's helpful.

Reply:
Fixed. Removed indicator for < iOS 11.